### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by utam0k**

### CODE REVIEW

File: crates/libcgroups/Cargo.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ procfs = "0.15.1"
 oci-spec = { version = "~0.6.3", features = ["runtime"] }
 fixedbitset = "0.4.2"
 serde = { version = "1.0", features = ["derive"] }
-rbpf = { version = "0.2.0", optional = true }
+rbpf = { version = "0.3.0", optional = true }
```
Comment (YJDoc2) [inline crates/libcgroups/Cargo.toml:28]: Is this accidental ? `rbpf` only has `0.2.0`

Comment (utam0k) [reply crates/libcgroups/Cargo.toml:28]: 🤦‍♂

File: justfile [hunk 2]
```diff
@@ -188,3 +188,8 @@ ci-musl-prepare: ci-prepare
 
     echo "Unknown system. The CI is only configured for Ubuntu. You will need to forge your own path. Good luck!"
     exit 1
+
+version-up version:
+    git grep -l "^version = .* # MARK: Version" | xargs sed -i 's/version = "[0-9]\.[0-9]\.[0-9]" # MARK: Version/version = "{{version}}" # MARK: Version/g'
+    sed -i s/_[0-9]_[0-9]_[0-9]_/_{{ replace(version, '.', '_') }}_/g  docs/src/user/basic_setup.md
+    sed -i 's/[0-9]\.[0-9]\.[0-9]/{{version}}/g'  docs/src/user/basic_setup.md
```
Comment (YJDoc2) [inline justfile:195]: :100:

**Comment by YJDoc2** [comment]
Hey @utam0k , I'll approve this PR, but before merge, can you check and merge https://github.com/containers/youki/pull/2438 ? It contains a simple bugfix for #2433 , would be nice if it also gets released along with this.

**Review by YJDoc2 — APPROVED** [review]
:tada: :tada: :tada:

**Pull request merged**